src/js/stores/__tests__/MesosStateStore-test.js

```diff
@@ -136,10 +247,81 @@ describe("MesosStateStore", function() {
       expect(result.fakeProp).toEqual("fake");
     });
 
-    it("returns null if node not found", function() {
+    it("returns undefined if node not found", function() {
       var result = MesosStateStore.getNodeFromID("nonExistentNode");
       expect(result).toEqual(undefined);
     });
+
+    it("returns null with invalid state", function() {
+      MesosStateStore.get = () => null;
+
+      var result = MesosStateStore.getNodeFromID("nonExistentNode");
+      expect(result).toEqual(undefined);
```

Comment:
You used `.toBeUndefined` above, use it here also?

Reply:
@ahoskins just fyi we do have cases where we use `.toEqual(undefined);` but this is too small to block a blocker